thesis-156/run-thesis-156.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+docker network create -d bridge --subnet=172.20.0.0/16 --gateway=172.20.0.1 thesis-156
+
+docker build --file THESIS-155/Dockerfile -t thesis-155:latest .
```

Comment:
@TanooKacha ขอเหตุผลหน่อยว่าทำไมต้องมีทั้ง `--file THESIS-155/Dockerfile` และ `.` ข้างหลัง

ปล. เคย test ด้วยการลบ image ทิ้งแล้ว รันคำสั่งนี้หรือไม่

Reply:
--file THESIS-155/Dockerfile  และ .   ข้างหลัง  อ้างอิง E book  ที่ให้มา

Reply:
@TanooKacha แล้วสรุปว่า ถ้า remove image ทิ้งแล้ว run ผ่านมั้ย ??

Reply:
```suggestion
docker build -t thesis-155:latest ../thesis-155/
```

Reply:
ก็ตามที่บอกไว้ว่า เคสปกติจะไม่นิยมทำ Dockerfile หลายอันใน directory เดียว
เพราะฉะนั้น...

1. แค่ path ของ Dockerfile ก็เจ๊งแล้ว ดันไปใช้ ตัวใหญ่ unix เป็นระบบ CASE-SENSITIVE ท่องไว้ให้ขึ้นใจ
2. ถ้าคิดจะใช้ `--file` ก็ระบุ Path ให้ถูก
ทำให้มีสองรูปแบบที่เป็นไปได้ ถ้าเรารัน `thesis-156$> ./run-thesis-156.sh` คือ
```suggestion
docker build -t thesis-155:latest ../thesis-155/
```
ถ้าห้าวจะใช้ `--file` ก็ระบุมันให้ถูกไปเลย จะไปไม่เป็นภาระลูกหลาน
```suggestion
docker build -t thesis-155:latest --file ../thesis-155/Dockerfile .
```
แต่เนื่องจากสุดท้ายปลายทางก็จุดอยู่ดี เราเลยเลือกวิธีแรกละกัน